app/assets/css/js-depend.css

```diff
@@ -4,4 +4,8 @@
 
 .js-enabled  .govuk-back-link{
  display: inline-block;
+}
+
+.autocomplete__input {
+    font-family: inherit;
```

Comment:
do you need this ?

Reply:
yes, as the autocomplete_input did not had a font family, now added a inherit property for it in the custom css renaming to application.css